### PR TITLE
LPS-133886 Add test to verify if the fields are searchable by the configuration property

### PR DESCRIPTION
**Pull request by marcelabc**
Related ticket: [LPS-133886](https://issues.liferay.com/browse/LPS-133886)

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by guilhermedcamacho** [comment]
Just started reviewing :)

File: modules/apps/data-engine/data-engine-taglib/src/test/java/com/liferay/data/engine/taglib/internal/servlet/taglib/util/DataLayoutTaglibUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+		JSONArray actualResultJSONArray =
+			_dataLayoutTaglibUtil.getFieldTypesJSONArray(
+				_httpServletRequest, Collections.singleton(_JOURNAL), false);
+
+		Assert.assertEquals(
+			_objectMapper.readTree(
+				read(
+					"data-definition-field-types-journal-scope-searchable-" +
+						"fields-enabled.json")),
```
Comment (guilhermedcamacho) [inline modules/apps/data-engine/data-engine-taglib/src/test/java/com/liferay/data/engine/taglib/internal/servlet/taglib/util/DataLayoutTaglibUtilTest.java:90]: Hey @marcelabc ,

Overall, your code looks good, I'm just a little confuse about why are you checking if the result is equals to "fields-**enabled**.json" in this method that handles the opposite "WithSearchableFields**Disabled**".

What do you think to change it to be more clear for other developers?

Let me know if I understood something wrong. Thanks!

File: modules/apps/data-engine/data-engine-taglib/src/test/java/com/liferay/data/engine/taglib/internal/servlet/taglib/util/DataLayoutTaglibUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+			_objectMapper.readTree(actualResultJSONArray.toString()));
+	}
+
+	@Test
+	public void testGetFieldTypesJSONArrayWithSearchableFieldsEnabled()
```
Comment (guilhermedcamacho) [inline modules/apps/data-engine/data-engine-taglib/src/test/java/com/liferay/data/engine/taglib/internal/servlet/taglib/util/DataLayoutTaglibUtilTest.java:95]: Same thing of last comment.

**Review by guilhermedcamacho — COMMENTED** [review]
Waiting for sender answer

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:forward

**Pull request closed without merging**